mlserver/server.py

```diff
@@ -22,7 +22,9 @@
 
 
 class MLServer:
-    def __init__(self, settings: Settings):
+    def __init__(
+        self, settings: Settings, model_repository: Optional[ModelRepository] = None
```

Comment:
To keep the interface simple, I would remove the `model_repository` kwarg and just rely on what comes from the `settings` object. This is also consistent with the rest of parameters and flags.

Reply:
Done